comtypes/GUID.py

```diff
@@ -8,6 +8,13 @@ def binary(obj):
     def binary(obj):
         return buffer(obj)
 
+if sys.version_info >= (3, 0):
+    text_type = str
+    base_text_type = str
+else:
+    text_type = unicode
+    base_text_type = basestring
```

Comment:
What if we move all these definitions into a settings/constants module and then we just import them from there everywhere we use them? (so we don't repeat with this branching everywhere and in case we want to change a type, we just change it in one place (as it's imported everywhere else is used))

Reply:
That's a good idea

Reply:
That's good idea, but the time is very limited. If someone will take care of it, I can review and merge it as separate PR. Currently I'm going to release 1.1.11 without these cosmetics.